src/HttpKernel/ContaoCache.php

```diff
@@ -10,11 +10,44 @@
 
 namespace Contao\ManagerBundle\HttpKernel;
 
+use FOS\HttpCache\SymfonyCache\CacheInvalidation;
+use FOS\HttpCache\SymfonyCache\EventDispatchingHttpCache;
```

Comment:
`friendsofsymfony/http-cache` must be added to `composer.json` because of this usage